networks/models.py

```diff
@@ -38,3 +39,42 @@ class Moderator(Model):
     class Meta:
         database = db
         db_table = 'moderators'
+
+
+#Twitter related tables
+'''
+twatter tables with many to many relations:
+create_tables([
+    Tooter,
+    DiscordServer,
+    DiscordServer.tooters.get_through_model(),
+    DiscordChannel,
+    DiscordChannel.tooters.get_through_model()  ])
+'''
+class Tooter(Model):
+    screen_name = CharField(unique=True)
+    last_tweet_id = IntegerField(default=0)
+    retweeted_id = IntegerField(default=0)
+
+    class Meta:
+        database = db
+        db_table = 'tooters'
+
+
+class DiscordServer(Model):
+    name = CharField()
+    tooters = ManyToManyField(Tooter, related_name='servers')
+
+    class Meta:
+        database = db
+        db_table = 'servers'
+
+
+class DiscordChannel(Model):
```

Comment:
Suggestions? If I move channel into the server table, whats the best practice to have multiple channel ids in the feild?